include/hdlConvertor/hdlObjects/hdlLibrary.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <hdlConvertor/hdlObjects/named.h>
+#include <hdlConvertor/hdlObjects/iHdlObj.h>
+
+namespace hdlConvertor {
+namespace hdlObjects {
+
+/*
+ * HDL library reference
+ * */
+class HdlLibrary : public WithNameAndDoc, public iHdlObj {
+public:
+	//rhinton:: Although the grammar allows for a list of library 
```

Comment:
That means that reading in a VHDL file and writing it back out won't produce identical lexical results.  If the user puts two libraries in a list in a single library clause, hdlConverter will write it back out as two clauses with a single library in each.  

I'm beginning to change my mind, though, to favor designing the `HdlLibrary` object to model the structure of the language instead of abstracted semantics.  Abstracting the semantics is fine for a "converter" application.  But if this project is to be useful as a more general parser front-end (as I hope it will be), the code structure should be followed quite closely.

Reply:
I would prefer the single name library statement as there are many ridiculous/deprecated/never used by anyone features in every SV/VHDL standard. If you see the value in implementation with vector<string> I am not against it, but I think it is not worth the time to support it exactly as it is in VHDL.

Reply:
I decided to take the path you suggested.  This is consistent with the way you're handling constant, variable, and signal declarations, interface lists, etc.  And I think it makes the AST simpler.  And the code as written does not decrease flexibility -- it can still handle multiple libraries in a single declaration.  It just transforms them to an easy-to-use form.